internal/controller/platform/standalone/nimservice.go

```diff
@@ -290,7 +290,7 @@ func (r *NIMServiceReconciler) isDeploymentReady(ctx context.Context, namespaced
 
 	cond := getDeploymentCondition(deployment.Status, appsv1.DeploymentProgressing)
 	if cond != nil && cond.Reason == "ProgressDeadlineExceeded" {
```

Comment:
Should we even have this check?

Reply:
The function is used to get the deployment status. I feel the more the details we have on NIMservice we have the better it is.